examples/federated-learning/tff/nvflare/docker-image/requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-network
+tensorflow
```

Comment:
Set fixed versions so we make the build more reproducible. Also, use hashes so we can check the integrity of these packages.

Reply:
Done